Add updated description of Just-In-Time Aggregates (#291)
timescale/docs.timescale.com-content: release-notes.md

### What to expect from our next release

The next release of TimescaleDB, v1.7, will include basic
[PostgreSQL 12](https://www.postgresql.org/about/news/1976/)
support and a new TimescaleDB capability called *Just-In-Time Aggregates*.

With Just-In-Time Aggregates, users will be able to leverage TimescaleDB's
precomputed continuous aggregates while still getting a real-time view over the
latest data.

In particular, continous aggregates made it really fast to query data:

> precompute some aggregate (such as the min/max/average value over each

Reviewer: Not sure what you are trying to convey here?  The grammar is odd.

> hour). This way, if you are collecting raw data every seconds, to query hourly

Reviewer: ```suggestion
hour). This way, if you are collecting raw data every second, querying hourly
```

> data over the past week means reading 24*7=168 values from the database, as

Reviewer: escape all of the asterisks here and line below to prevent odd visuals:
![Screen Shot 2020-02-14 at 12 45 54 PM](https://user-images.githubusercontent.com/12288273/74554633-f9b61500-4f27-11ea-9ddb-0a253755b3c3.png)

Reviewer: or switch to 'x'

opposed to processing 60*60*24*7=604,800 values at query time.  But one

> limitation with continous aggregates is they don't incorporate the very latest

Reviewer: ```suggestion
limitation with continous aggregates is that they don't incorporate the very latest
```

> data, i.e., since the asynchronous aggregation job last ran inside the

Reviewer: ```suggestion
data, _i.e._, since the last time the asynchronous aggregation job ran inside the
```
parses more clearly?

database. So if you are generating hourly rollups, you might only run this
job every hour.

But now with Just-In-Time Aggregates, a single, simple query will incorporate your
pre-computed hourly rollups while aggregating just the raw data from the last

> hour, to always give you an up-to-date answer.  Now, instead of touching

Reviewer: Consider:
> ...a single, simple query will combine your pre-computed hourly rollups with the raw data from the last hour...

### What to expect from our next release

The next release of TimescaleDB, v1.7, will include basic
[PostgreSQL 12](https://www.postgresql.org/about/news/1976/)
support and a new TimescaleDB capability called *Just-In-Time Aggregates*.

With Just-In-Time Aggregates, users will be able to leverage TimescaleDB's
precomputed continuous aggregates while still getting a real-time view over the
latest data.

> In particular, continous aggregates made it really fast to get aggregate 

Reviewer 2: typo continous

answers by precomputing these values (such as the min/max/average value over
each
hour). This way, if you are collecting raw data every second, querying hourly
data over the past week means reading 24 x 7 = 168 values from the database, as
opposed to processing 60 x 60 x 24 x 7 = 604,800 values at query time.  But one

> limitation with continous aggregates is that they don't incorporate the very

Reviewer 2: "continous" typo here.

latest data, _i.e._, since the last time the asynchronous aggregation job ran
inside the database. So if you are generating hourly rollups, you might only
run this job every hour.

But now with Just-In-Time Aggregates, a single, simple query will combine your
pre-computed hourly rollups with the raw data from the last
hour, to always give you an up-to-date answer.  Now, instead of touching
604,800 rows of raw data, the query reads 167 pre-computed rows of

> hourly data and 3600 rows of raw secondly data, leading to significant

Reviewer 2: "raw secondly data" typo

this table.

We added a timescaledb.ignore_invalidation_older_than parameter for continuous

> aggregates. This parameter accept a time-interval (e.g. 1 month). If set,

Reviewer 2: accepts
